test/unit/openapi/streaming/subscription-spec.js

```diff
@@ -807,20 +814,19 @@ describe("openapi StreamingSubscription", () => {
             sendInitialResponse({InactivityTimeout: 100, Snapshot: { resetResponse: true }});
 
             tick(() => {
-                const firstArgs = { newArgs: 'firstArgs' };
-                const secondArgs = { newArgs: 'secondArgs' };
-                subscription.onModify(firstArgs, { isPatch: true });
-                subscription.onModify(secondArgs, { isPatch: true });
+                const newArgs = { newArgs: 'test' };
```

Comment:
this is changing the test - shouldn't it pass the same thing as before - and we test that the last args is used?